src/editor/PositionPlacementControl.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ButtonGroup, Button, Tooltip } from '@wordpress/components';
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+const PositionPlacementControl = ( { layout, label, help, onChange, size, ...props } ) => {
+	/**
+	 * Set layout options
+	 */
+	const layoutOptions =
```

Comment:
IMHO the right way to do this would be to reset the placement whenever the site is set to full-width based on the previous placement value (e.g. if it was `center-left`, the new value would be `center`) the drawback to this would be the loss of the previous value, continuing on the same example if we select back a non-full-width size the position will still `center` and won't go back to `center-left`.

A hacky way to do this is to _fake_ the position selected by not relying only on `value === layout` but adding conditions depending on the size. Thing that should be considered in the backend too.

I think the first approach is safe and clean here, but I would like to have your inputs on this :)

Reply:
> IMHO the right way to do this would be to reset the placement whenever the site is set to full-width based on the previous placement value

Yeah, makes sense 👌 

> the drawback to this would be the loss of the previous value

I don't think this is much of a concern, this is a pretty simple setting.

Reply:
Thank you @adekbadek